tests/components/nest/test_media_source.py

```diff
@@ -875,7 +875,7 @@ async def test_event_media_render_invalid_event_id(hass, auth, hass_client):
     assert device.name == DEVICE_NAME
 
     client = await hass_client()
-    response = await client.get("/api/nest/event_media/{device.id}/invalid-event-id")
+    response = await client.get(f"/api/nest/event_media/{device.id}/invalid-event-id")
```

Comment:
This test needs to be fixed, it was probably passing since the missing f string cause it to return 404.
I suggest to remove this from this PR and handle this in a separate PR

Reply:
Fixing in https://github.com/home-assistant/core/pull/70596 and you can revert from this PR, thanks!